selfdrive/car/honda/carcontroller.py

```diff
@@ -178,6 +178,9 @@ def update(self, enabled, CS, frame, actuators, \
         idx = frame // 2
         ts = frame * DT_CTRL
         pump_on, self.last_pump_ts = brake_pump_hysteresis(apply_brake, self.apply_brake_last, self.last_pump_ts, ts)
+        # If pedal detected, don't send cancel command to prevent brake grinding.
```

Comment:
I don't think this is the right place for this, it's way too close to the output. Why is cancel being generated, and is there a better place to block it if that's what we want.